pkg/controllers/nodeclass/status/securitygroup.go

```diff
@@ -50,5 +50,5 @@ func (sg *SecurityGroup) Reconcile(ctx context.Context, nodeClass *v1beta1.EC2No
 			Name: *securityGroup.GroupName,
 		}
 	})
-	return reconcile.Result{RequeueAfter: 5 * time.Minute}, nil
+	return reconcile.Result{RequeueAfter: time.Minute}, nil
```

Comment:
This is going to increase our call rate across everything right now. What's our impact on call volume with this change? We're calling 5 times as often but we haven't taken things like discover subnets out of the critical path so what's the impact?

Reply:
Decided to leave this as 5 minutes